src/ansys/fluent/core/launcher/slurm_launcher.py

```diff
@@ -185,43 +205,164 @@ class SlurmLauncher:
+        start_transcript : bool, optional
+            Whether to start streaming the Fluent transcript in the client. The
+            default is ``True``. You can stop and start the streaming of the
+            Fluent transcript subsequently via the method calls, ``transcript.start()``
+            and ``transcript.stop()`` on the session object.
+        case_file_name : str, optional
+            Name of the case file to read into the
+            Fluent session. The default is ``None``.
+        case_data_file_name : str, optional
+            Name of the case data file. If names of both a case file and case data file are provided, they are read into the Fluent session.
+        lightweight_mode : bool, optional
+            Whether to run in lightweight mode. In lightweight mode, the lightweight settings are read into the
+            current Fluent solver session. The mesh is read into a background Fluent solver session which will
+            replace the current Fluent solver session once the mesh read is complete and the lightweight settings
+            made by the user in the current Fluent solver session have been applied in the background Fluent
+            solver session. This is all orchestrated by PyFluent and requires no special usage.
+            This parameter is used only when ``case_file_name`` is provided. The default is ``False``.
+        py : bool, optional
+            If True, Fluent will run in Python mode. Default is None.
+        gpu : bool, optional
+            If True, Fluent will start with GPU Solver.
+        cwd : str, Optional
+            Working directory for the Fluent client.
+        topy : bool or str, optional
+            A boolean flag to write the equivalent Python journal(s) from the journal(s) passed.
+            Can optionally take the file name of the new python journal file.
+        start_watchdog : bool, optional
+            When ``cleanup_on_exit`` is True, ``start_watchdog`` defaults to True,
+            which means an independent watchdog process is run to ensure
+            that any local GUI-less Fluent sessions started by PyFluent are properly closed (or killed if frozen)
+            when the current Python process ends.
+        scheduler_options : dict, optional
+            Dictionary containing scheduler options. Default is None.
+
+            Currently only the Slurm scheduler is supported. The ``scheduler_options``
+            dictionary must be of the form ``{"scheduler": "slurm",
+            "scheduler_headnode": "<headnode>", "scheduler_queue": "<queue>",
+            "scheduler_account": "<account>"}``. The keys ``scheduler_headnode``,
+            ``scheduler_queue`` and ``scheduler_account`` are optional and should be
+            specified in a similar manner to Fluent's scheduler options.
+        file_transfer_service : optional
+            File transfer service. Uploads/downloads files to/from the server.
```

Comment:
```suggestion
        file_transfer_service : optional
            File transfer service for uploading and downloading files to and from the server.
```
Description should be in simple English, avoiding the slash constructions.

Reply:
Done. Thank you.